frontend/src/pages/Login.tsx

```diff
@@ -24,9 +24,8 @@ const Wrapper = styled.div`
   align-items: center;
   gap: 13.67vh;
   padding: 20vh 364.6px 0vh 365px;
-  // max-height: 70vh;
 `;
-const Button = styled.button`
+export const Button = styled.button`
```

Comment:
For now, I didn't make Button.tsx a full fledged component class, just a few exported styled components; is that alright? I couldn't quite figure out how to make it a component that accepts props that can also be further modifed by styled components in different classes (because from what I've seen on Figma not all the button designs are consistent).

Reply:
That's fine. If we find we need to consolidate into one component, we can refactor down the line.